app.js

```diff
@@ -1,7 +1,99 @@
+const data = [
+  {
+    heading: "Tonic",
+    image: "./assets/project1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+    and typesetting industry. Lorem Ipsum has been the industry's 
+    standard dummy text ever since the 1500s, when an unknown 
+    printer took a galley of type and scrambled it to make a type 
+    specimen book. It has survived not only five centuries, but 
+    also the leap into electronic typesetting, remaining essent`,
```

Comment:
- [ ] You should add these two example attributes as you are gonna need them in your future job search. (You are going to use different languages and source links for each of your project )
```suggestion
   * technologies: ["Phaser3", "JavaScript", "Jest"],
   * link: "link_to_live_version.com",
```